index.html

```diff
@@ -23,12 +25,11 @@
         </ul>
       </nav>
     </header>
-
-
-    <section class="headline">
-      <h1 class="title">
-         I'm Luis <br>Glad to see you!
-      </h1>
+    <div class="headline-background">
+      <section class="headline">
+        <h1 class="title">
+          I'm Luis <br>Glad to see you!
+        </h1>
         <div class="header-content">
           <p class="descr">I’m a software developer! I can help you build a product, feature or website. Look through
```

Comment:
-[OPTIONAL] Kindly add some margin to this section and make the paragraph responsive for mobile devices.
![Captura de pantalla_2021-09-22_17-56-19](https://user-images.githubusercontent.com/53111464/134432709-fdc9d266-446d-4dfa-bf3f-cd1f25576bca.png)